scripts/r.in.wms/wms_drv.py

```diff
@@ -34,7 +34,8 @@
 
 Numeric.arrayrange = Numeric.arange
 
-from math import pi, floor
+# code changed here noqa E402 added
+from math import pi, floor  # noqa: E402
```

Comment:
Here deactivating the flake check might be appropriate. We have lazy imports also at other places, unsure how it is handeled there....